src/Microsoft.Data.Sqlite/SqliteConnectionStringBuilder.cs

```diff
@@ -14,27 +14,32 @@ namespace Microsoft.Data.Sqlite
     public class SqliteConnectionStringBuilder : DbConnectionStringBuilder
     {
         private const string DataSourceKeyword = "Data Source";
+        private const string CacheKeyword = "Cache";
 
         private enum Keywords
         {
-            DataSource
+            DataSource,
+            Cache
         }
 
         private static readonly IReadOnlyList<string> _validKeywords;
         private static readonly IReadOnlyDictionary<string, Keywords> _keywords;
 
         private string _dataSource = string.Empty;
+        private CacheMode _cacheMode = CacheMode.Private;
```

Comment:
We will defer adding a `Default` mode until we modify how connections are opened. (switching to open_v2)